src/hexen/p_user.c

```diff
@@ -237,7 +237,7 @@ void P_MovePlayer(player_t * player)
         }
         else
         {
-            P_Thrust(player, player->mo->angle, FRACUNIT >> 8);
```

Comment:
I'm pretty sure we need to keep this for bug compatibility (read: demo compatibility) with Vanilla Hexen. But Doom never had jumping or in-air movement, so the change is okay there.